plotter/general_test.go

```diff
@@ -81,6 +79,7 @@ func checkPlot(ExampleFunc func(), t *testing.T, filenames ...string) {
 
 // Draw the plot logo.
 func Example() {
+
```

Comment:
I'm assuming "these" refers to empty lines at the beginning of functions? I will delete them.